Update container plugin from 0.6.1 to 0.6.3 to fix GHSA-fcv2-xgw5-pqxf

The container plugin v0.6.3 includes an update to github.com/sigstore/sigstore
from v1.9.5 (vulnerable) to v1.10.4 (fixed), which resolves CVE-2026-24137 /
GHSA-fcv2-xgw5-pqxf.

This vulnerability is a path traversal issue in the sigstore legacy TUF client
that allows for arbitrary file writes with target cache path traversal.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -282,11 +282,11 @@ add_subdirectory(userspace/falco)
    AND NOT MUSL_OPTIMIZED_BUILD
 )
 	include(falcoctl)
-	set(CONTAINER_VERSION "0.6.1")
+	set(CONTAINER_VERSION "0.6.3")
 	if(${CMAKE_HOST_SYSTEM_PROCESSOR} STREQUAL "x86_64")
-		set(CONTAINER_HASH "008989992ed1f31b3ffb94ba6b64ca5a8e2f91611a10c9d6213c5c0a499d0679")
+		set(CONTAINER_HASH "dd5e92628fbdc192a1a8f13aafaba4c673e9cba268daec66480403f8a8c4590d")
 	else() # arm64
-		set(CONTAINER_HASH "f90a700b4c2b411b23e7cc461b61a316b242994aad853c3e6baf12481fb6f6c9")
+		set(CONTAINER_HASH "1170f1ad10863d37351821532b42da3ca19090b490c11d7d7a31cdebe522b3f5")
 	endif()
 	include(container_plugin)

